docs/cmdlets-reference/restore-cohesityremotefile.md

```diff
@@ -1,100 +1,91 @@
 # Restore-CohesityRemoteFile
 
 ## SYNOPSIS
-Restores the specified files or folders from a remote cluster.
+Restores the specified files or folders from a previous remote backup based on Cohesity V2 Rest APIs.
 
 ## SYNTAX
 
 ```
-Restore-CohesityRemoteFile [[-TaskName] <String>] [-FileNames] <String[]> [-JobId] <Int64> [-SourceId] <Int64>
- [[-NewBaseDirectory] <String>] [[-JobRunId] <Int64>] [[-StartTime] <Int64>] [-DoNotOverwrite]
- [-ContinueOnError] [-DoNotPreserveAttributes] [-TargetSourceId] <Int64>
- [[-TargetHostCredential] <PSCredential>] [-WhatIf] [-Confirm] [<CommonParameters>]
+Restore-CohesityFileV2 [-TaskName <String>] [-FileName <String>] [-JobId <Int64>] [-SourceId <Int64>]
+    [-TargetSourceId <Int64>] [-NewBaseDirectory <String>] [-SnapshotId <String>] [-OverwriteExisting] 
+    [-ContinueOnError] [-EncryptionEnabled] [-PreserveAttributes] [-SaveSuccessFiles] [-RecoverMethod <String>]
+    [-TargetVMCredential <PSCredential>] [-WhatIf] [-Confirm] [<CommonParameters>]
 ```
 
 ## DESCRIPTION
-Request to create a Restore Task for recovering files or folders from a remote target.
+Request to create a task for recovering the specified files or folders from a previous remote backup based on Cohesity V2 Rest APIs.
+This commandlet supports only source with environment type VMware/Physical/Isilon.
 
 ## EXAMPLES
 
 ### EXAMPLE 1
 ```
-Restore-CohesityRemoteFile -TaskName "restore-file-vm" -FileNames /C/data/file.txt -JobId 1234 -SourceId 843 -TargetSourceId 856 -TargetParentSourceId 828 -TargetHostCredential (Get-Credential)
+Restore-CohesityRemoteFile -TaskName "restore-file-vm" -FileName /C/data/file.txt -JobId 1234 -SourceId 843 -TargetSourceId 856 -RestoreMethod AutoDeploy -TargetVMCredential (Get-Credential)
+        
 ```
 
-Restores the file from the specified source to the target windows VM using the latest backup from remote target.
-
 ### EXAMPLE 2
 ```
-Restore-CohesityRemoteFile -TaskName "restore-file-vm" -FileNames /C/data/file.txt -JobId 1234 -JobRunId 3005 -StartTime 1690646467987573 -SourceId 843 -TargetSourceId 856 -TargetParentSourceId 828 -TargetHostType KWindows -TargetHostCredential (Get-Credential)
```

Comment:
We aren't deprecating this correct ? just adding another commandlet with right usecase ?

Reply:
We are using the name same for the commandlet to restore remote file using v2 api now. Previously this commandlet uses private v1 api.